goalie/go_mesh_seq.py

```diff
@@ -65,6 +68,48 @@ def forms(self):
             )
         return self._forms
 
+    def _reset_changing_coefficients(self):
+        self._init_form_coeffs = None
+        self._changed_form_coeffs = {field: {} for field in self.fields}
```

Comment:
TODO: describe that `self._changed_form_coeffs` looks like `{field: {coeff_idx: {export_timestep_idx: coefficient}}}`, where `coeff_idx` is the index of `coefficient` in `F.coefficients()`

Reply:
Done in https://github.com/mesh-adaptation/goalie/pull/259/commits/abdb7ce66aeabf45302e287181e721a9a7054fc7